experiments/check_env_sanity.cfg

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
```

Comment:
This file shouldn't be called "*.cfg". This is a commonly used extension for config files, having a script with that name is going to confuse people.

Reply:
Agreed. It started life as only having some variables, but it outgrew this use case, without an appropriate rename following it.